Appsilon/shiny.fluent
# Fix demo app and update package dependencies

### Changes
Closes #195

### How to test

Install the package: 
```r
remotes::install_github("Appsilon/shiny.fluent@feat/195-fix-demo-app", dependencies = "Suggests")
```

check that demo app is working:

```r
shiny.fluent::runExample("demo")
```
